src/repl/repl.cpp

```
@@ -263,6 +263,10 @@ int run_repl(ChatFn chat, const Config& cfg, std::istream& in, std::ostream& out
   std::vector<Message> history;
   if (!cfg.system_prompt.empty()) {
     std::string sys = cfg.system_prompt;
```

Clang (14.0.6)

[warning] variable 'sys' is not initialized

(cppcoreguidelines-init-variables)